### PR TITLE
[Keyboard] Add support for Alveus MX PCB 

### PR DESCRIPTION
**Pull request by mechlovin**
## Description
Add support for Alveus MX version. (The EC version by Cipulot).
Thanks.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/densus/alveus/mx/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        KC_LCTL,   KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RALT, MO(1),  KC_RCTL,            KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    [1] = LAYOUT_ansi(
+        KC_TRNS,  KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,            KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,            KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,                                      
+        KC_TRNS,            KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,                     KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,                            KC_TRNS,                                     KC_TRNS, KC_TRNS, KC_TRNS,             KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+
```
Comment (waffle87) [inline keyboards/densus/alveus/mx/keymaps/default/keymap.c:33]:
```suggestion
```

File: keyboards/densus/alveus/mx/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        KC_TRNS,  KC_TRNS,  KC_TRNS,                            KC_TRNS,                                     KC_TRNS, KC_TRNS, KC_TRNS,            KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+    [3] = LAYOUT_all(
+        KC_TRNS,  KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,            KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                       
+        KC_TRNS,  KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                     KC_TRNS,
+        KC_TRNS,  KC_TRNS,  KC_TRNS,                            KC_TRNS,                                     KC_TRNS, KC_TRNS, KC_TRNS,            KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+
```
Comment (waffle87) [inline keyboards/densus/alveus/mx/keymaps/via/keymap.c:47]:
```suggestion
    )
```

File: keyboards/densus/alveus/mx/info.json [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+    "matrix_pins": {
+        "rows": ["B12", "B13", "B14", "A8", "A2"],
+        "cols": ["B10", "B2", "B1", "B0", "A7", "A6", "A5", "A4", "A3", "A15", "B9", "B8", "B7", "B6", "B5", "B4", "B3"]
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": true,
+        "command": true,
```
Comment (waffle87) [inline keyboards/densus/alveus/mx/info.json:20]: Is there a reason these two are enabled by default?

Comment (mechlovin) [reply keyboards/densus/alveus/mx/info.json:20]: Hi! No particular reason for this. I always enable these on all of my PCBs. Please let me know if there are any implications that I need to consider regarding these.

Comment (waffle87) [reply keyboards/densus/alveus/mx/info.json:20]: You can leave them enabled if you'd like, however they're not particularly useful for most people, increase firmware size, and can negatively impact other performance aspects such as matrix scan rate.

**Review by waffle87 — APPROVED** [review]

**Pull request merged**